src/utils/api/api.js

```diff
@@ -8,6 +8,9 @@ async function api(endpoint, init = {}) {
     case 'roles':
       url = '/jsonapi/user_role/user_role';
       break;
+    case 'content':
```

Comment:
So yeah we miss the feature to fetch nodes from multiple bundles here :(

Reply:
#96 would allow this!